.gitignore

```diff
@@ -26,6 +26,7 @@ builder/rootfs/usr/bin
 cache/image/bin/
 client/dist/
 client/makeself/
+client-go/deis
```

Comment:
I'd make it replace the `client/` dir entirely. We can archive the python client and move it elsewhere for historical purposes.